plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/CompareViewImpl.java

```diff
@@ -45,29 +47,36 @@
     private static final PreviewViewImplUiBinder UI_BINDER = GWT.create(PreviewViewImplUiBinder.class);
 
     @UiField
-    SimplePanel comparePanel;
+    DockLayoutPanel dockPanel;
```

Comment:
Try to avoid using DockLayout panel in future. Use FlowPanel instead. It will allow you to simple support in styling.